thoth-client/src/parameters.rs

```diff
@@ -294,6 +300,8 @@ mod tests {
                 with_fundings: false,
                 with_relations: false,
                 with_references: false,
+                with_jats_xml_abstracts: false,
```

Comment:
These ought to have corresponding `.without_jats_xml_abstracts()` and `.without_plain_text_abstracts()` set on the `QueryParameters::new()` object (ditto below). You also need to define these `.with...()` and `.without...()` functions in the `impl QueryParameters` block, following the existing pattern.